### PR TITLE
add common back btn, add purchased courses to student dashboard

### PR DESCRIPTION
**Pull request by rawrrencee**
# Feature
Common back button component
Skeleton code for Purchased courses/view purchased course (I just realised this is under D and not C, so I will be closing this branch after this PR)

Use Case: D.2.5 View Purchased Courses
Feature ID : 8
Feature: Course Content

## Checklist:

- [x] Merged latest develop
- [x] PR title makes sense

## Screenshots (where relevant):
image

### CODE REVIEW

File: src/pages/student/dashboard/courses/view/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import BackBtn from 'components/Common/BackBtn'
+import React from 'react'
+import { useParams } from 'react-router-dom'
+
+const StudentCourseDetails = () => {
+  const { id } = useParams()
+  console.log(id)
```
Comment (natalieloke) [inline src/pages/student/dashboard/courses/view/index.js:7]: extra console log to remove in future

File: src/pages/student/dashboard/courses/view/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <div>
+      <div className="row pt-2">
+        <div className="col-12 col-md-3 col-lg-2 mt-4 mt-md-0">
+          <BackBtn />
+        </div>
+      </div>
+      <span>View Course</span>
+    </div>
```
Comment (natalieloke) [inline src/pages/student/dashboard/courses/view/index.js:16]: ps can I see a ss of how the back button looks like on this page

File: src/pages/student/dashboard/courses/index.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+const StudentPurchasedCourses = () => {
+  const user = useSelector(state => state.user)
+
+  const [courses, setCourses] = useState([])
+  const [isLoading, setIsLoading] = useState(false)
+
+  const getAllPurchasedCourses = async () => {
+    setIsLoading(true)
+    const result = await getPurchasedCourses(user.accountId)
+    console.log('result', result)
```
Comment (natalieloke) [inline src/pages/student/dashboard/courses/index.js:19]: extra console log

**Review by natalieloke — APPROVED** [review]

File: src/pages/student/dashboard/courses/view/index.js (hunk 2 above)
Comment (rawrrencee) [reply src/pages/student/dashboard/courses/view/index.js:16]: sure!!

image
image

**Pull request merged**